src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -708,1364 +582,1200 @@
-
-      <div class="bs-example" id="preview">
-        <div class="well well-plain" id="questionPreviewTable">
-          <div class="row">
-            <form class="form-horizontal">
-              <label class="control-label col-sm-2 text-right">
-                Preview Session:
-              </label>
-            </form>
-            <div class="col-sm-5" data-toggle="tooltip" data-placement="top" title="View how this session would look like to a student who is submitting feedback.<br>Preview is unavailable if the course has yet to have any student enrolled.">
-              <form name="form_previewasstudent" class="form_preview">
-                <div class="col-sm-6">
-                  <select class="form-control" name="previewas">
-                    <option value="alice.b.tmms@gmail.com">[Team 1] Alice Betsy</option>
-                    <option value="benny.c.tmms@gmail.com">[Team 1] Benny Charles</option>
-                    <option value="danny.e.tmms@gmail.com">[Team 1] Danny Engrid</option>
-                    <option value="emma.f.tmms@gmail.com">[Team 1] Emma Farrell</option>
-                    <option value="charlie.d.tmms@gmail.com">[Team 2] Charlie Davis</option>
-                    <option value="francis.g.tmms@gmail.com">[Team 2] Francis Gabriel</option>
-                    <option value="gene.h.tmms@gmail.com">[Team 2] Gene Hudson</option>
-                  </select>
-                </div>
-                <div class="col-sm-6">
-                  <input id="button_preview_student" class="btn btn-primary" value="Preview as Student">
-                </div>
-              </form>
-            </div>
-            <div class="col-sm-5" data-toggle="tooltip" data-placement="top" title="View how this session would look like to an instructor who is submitting feedback.">
-              <form class="form_preview">
-                <div class="col-sm-6">
-                  <select class="form-control" name="previewas">
-                    <option value="inst@gmail.com">Instructor A</option>
-                  </select>
-                </div>
-                <div class="col-sm-6">
-                  <input id="button_preview_instructor" class="btn btn-primary" value="Preview as Instructor">
-                </div>
-              </form>
-            </div>
-          </div>
-        </div>
-        <div>
-        </div>
-      </div>
-    </li>
-    <br>
-    <br>
-    <li id="fbViewResults">
-      <span class="text-bold">
-          <b>4. Viewing results</b>
-        </a>
-      </span>
-
-      <p>
-        You can view results of sessions by clicking the
-        <b>View Results</b> action of a session
-        <br>You can see responses in several different formats using the following controls.
-        <br>
-        <br> There are
-        <b>5</b> different views available, which indicate the order in which responses are grouped. Additionally, there are also options to
-        <b>group the results by tea
```

Comment:
This was missing out in previous PRs. Could you help to add in students (excluding self), and teams (excluding self)?

Reply:
What do you mean?